resources/js/utils/beatmap-helper.ts

```diff
@@ -99,6 +101,14 @@ export function group<T extends BeatmapJson>(beatmaps?: T[] | null, includeEmpty
   return ret;
 }
 
+export function hasGuestOwners(beatmap: WithBeatmapOwners<BeatmapJson>, beatmapset: BeatmapsetJson) {
+  return beatmap.owners.some((owner) => owner.id !== beatmapset.user_id);
+}
+
+export function isOwner(userId: number, beatmap: WithBeatmapOwners<BeatmapJson>) {
```

Comment:
there's at least another one missing usage of this in `nominator.tsx` (nominate button permission)